resources/views/home.blade.php

```diff
@@ -15,7 +15,7 @@
   <p class="home-welcome-title">Welcome, {{ $name }}.</p>
   <ul class="home-bar">
     <li><button onclick="window.location = '/logout' ">Logout</button></li>
-    <li><button onclick="window.location = '/user-settings' ">Settings</button></li>
+    <li><button onclick="window.location = '/user-settings?/home' ">Settings</button></li>
```

Comment:
what does this added `?/home` do?

Reply:
It is the variable that is being passed onto the user setting page.
I used method 2 shown by this website:
https://lage.us/Javascript-Pass-Variables-to-Another-Page.html

Reply:
Ah, gotcha. Clever, I like it!